srf_generation/source_parameter_generation/generate_realisations_from_gcmt.py

```diff
@@ -48,20 +49,23 @@ def load_args(primary_logger: Logger):
     parser.add_argument("fault_selection_file", type=abspath)
     parser.add_argument("gcmt_file", type=abspath)
     parser.add_argument("--version", type=str, default="unperturbated")
-    parser.add_argument("--vel_mod_1d", type=abspath)
+    parser.add_argument(
+        "--vel_mod_1d",
+        type=abspath,
+        default=join(dirname(__file__), "velocity_model", "lp_generic1d-gp01_v1.vmod"),
```

Comment:
since this default value is shared, is possibly a good idea to import it from one location (plus this script is importing other variables already, make sense to leave default values there)